.github/workflows/apk-release.yml

```diff
@@ -1,7 +1,7 @@
 # This workflow will build a Java project with Gradle
 # For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
 
-name: APK Release
+name: Quest APK Release
```

Comment:
What happens to a non Quest APK release ? in the case of Malawi core?

Reply:
Depends if we expect that as a recurring pattern i.e. creating new modules outside of Quest. 

We can implement a solution once we add the Malawi core module